app/controllers/health_check_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# HealthCheckController: allows user to check the service health status
```

Comment:
You are free to disable the comments for each class and each method - they are useless in most cases.

Write 'self-descriptional' code